student.rb

```diff
@@ -1,6 +1,8 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] There is no need to change the `classroom` instance variable from outside the Student class. therefor please use the attribute reader for this variable.